### PR TITLE
Asirota/add issue template

### PR DESCRIPTION
**Pull request by asirota**
# Pull Request

## What changed?

added ISSUE_TEMPLATE for GitHub

## Why did it change?

make it easier for new contributors to file bugs and feature requests

## Did you fix any specific issues?

fixed readme.txt license to GPLv2
#100 

## CERTIFICATION

By opening this pull request, I do agree to abide by
the [CODE OF CONDUCT](https://github.com/aspirepress/.github/CODE_OF_CONDUCT.md) and be bound by the terms
of the [Contribution Guidelines](https://github.com/aspirepress/.github/CONTRIBUTING.md) in effect on the date and time
of my contribution as proven by the
revision information in GitHub. I also agree that any previous contributions shall be deemed subject to the terms of the
version in effect on the date and time of this pull request, or any future revisions for pull requests I may submit.
Further, I certify that this work is my own, is original, does not violate the intellectual property of any other person
or entity, and I am not violating any license agreements or contracts I have with any person or entity. Finally, I agree
that this code may be licensed under any license deemed appropraite by AspirePress, including but not
limited to open source, closed source, proprietary or custom licenses, and that such license terms neither violate my
rights or my copyright to this code.

### CODE REVIEW

**Comment by costdev** [comment]
The plugin file's license is GPLv3 or later.  How come the license is changed to GPLv2 here?

**Comment by afragen** [comment]
May want to consider using the `Test Reports` plugin for some of this templating.

**Comment by asirota** [comment]
@costdev because it's gpl2. The readme.txt was a typo. It's not gpl3

@afragen I have no idea what you are describing. Link please ?

**Comment by costdev** [comment]
@asirota `aspire-plugin.php` and `composer.json` (therefore `composer.lock` - update locally after changing `.json` with `composer update`) state GPLv3 or later

**Comment by asirota** [comment]
That's wrong. @sarah-savage stated it is gplv2.

**Comment by afragen** [comment]
https://wordpress.org/plugins/test-reports/

**Comment by asirota** [comment]
I see now! Great work this is a good idea to have something like this. Will this be part of core at some point since this plugin appears to be part of a WordPress contribution team?

In any case it would be actually quite cool to leverage this work directly from our Plugin if it ever is part of core. For now I'll look at the issue templates. I assume you don't have yml files for these do you?

**Comment by afragen** [comment]
The plugin was from @costdev and me. 😉

**Comment by costdev** [comment]
Indeed - myself and @afragen originally built this into the official WordPress Beta Tester plugin, following work I'd done along with others in the WordPress Test Team to develop consistent bug/issue reproduction/patch testing reports based on our QA experience. After me and Andy integrated this in Beta Tester, we saw an uptick in consistent report formats and details from testers. This made it easier for contributors to spot differences in environments, configurations and user flows which in turn helped to identify the cause of bugs, or spot missed edge cases in patches.

After we both found it to have value as a standalone plugin, we detached it from Beta Tester and added it to the repo. As Andy says, it's more likely to become a canonical plugin than be integrated into Core.

**Comment by costdev** [comment]
Here's a sample of the Bug Report template, partially generated by ChatGPT:

```yml
name: Bug Report
description: Report a bug in the system
title: "[BUG] "
labels: bug
assignees: ''

body:
  - type: textarea
    id: description
    attributes:
      label: 'Description'
      description: 'Describe the bug.'
  
  - type: input
    id: wordpress
    attributes:
      label: 'WordPress'
      description: 'The version of WordPress that your site uses.'
  
  - type: input
    id: php
    attributes:
      label: 'PHP'
      description: 'The version of PHP your site runs on.'
  
  - type: input
    id: server
    attributes:
      label: 'Server'
      description: 'The server your site runs on (apache, nginx, etc).'
  
  - type: input
    id: database
    attributes:
      label: 'Database'
      description: 'The database engine used by your site.'
  
  - type: input
    id: browser
    attributes:
      label: 'Browser'
      description: 'The browser you use (Chrome, Firefox, etc).'
  
  - type: input
    id: os
    attributes:
      label: 'OS'
      description: 'The operating system your site runs on (Linux, Windows, Mac, etc).'
  
  - type: input
    id: theme
    attributes:
      label: 'Theme'
      description: 'The currently active theme.'
  
  - type: input
    id: must_use_plugins
    attributes:
      label: 'Must-Use Plugins'
      description: 'A list of Must-Use Plugins.'
  
  - type: input
    id: plugins
    attributes:
      label: 'Plugins'
      description: 'A list of active plugins.'
  
  - type: textarea
    id: steps_to_reproduce
    attributes:
      label: 'Steps to Reproduce'
  
  - type: textarea
    id: expected_results
    attributes:
      label: 'Expected Results'
      description: 'What should happen.'
  
  - type: textarea
    id: actual_results
    attributes:
      label: 'Actual Results'
      description: 'What actually happened.'
  
  - type: textarea
    id: additional_notes
    attributes:
      label: 'Additional Notes'
      description: 'Any additional details worth mentioning.'

```

File: .github/ISSUE_TEMPLATE/BugReport.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    attributes:
+      label: Screenshots/Live demo link
+      description: Add some screenshots to help explain the problem.
+    validations:
+      required: false
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow the AsiprePress [Code of Conduct](https://github.com/aspirepress/AspireUpdate?tab=coc-ov-file).
```
Comment (costdev) [inline .github/ISSUE_TEMPLATE/BugReport.yml:64]:
```suggestion
      description: By submitting this issue, you agree to follow the AspirePress [Code of Conduct](https://github.com/aspirepress/AspireUpdate?tab=coc-ov-file). 

```

**Pull request merged**